src/transactions.ts

```diff
@@ -174,6 +184,19 @@ export function actionFromRLP(sdk: SDK, rlp: Buffer): Action {
                 deposit: decodeU64(decoded[1]),
                 metadata: decoded[2]
             };
+        case CHANGE_PARAMS_ACTION_ID:
+            if (decoded.length <= 3) {
+                throw new Error(
+                    "A length of a RLP list of a changeParams action should be more than 3"
+                );
+            }
+            const [, , , signatures] = decoded;
```

Comment:
The layout of `decoded` is `[TAG, metadataSeq, params, sig1, sig2, sig3, ...]`
You need to triple-dot operator to assign all signatures to `signatures`
```js
const [, , , ...signatures] = decoded;
```
or just
```js
const signatures = decoded.slice(3);
```